spec/lcm/integration/support/lcm_helper.rb

```diff
@@ -0,0 +1,41 @@
+class LcmHelper
+  class << self
+    def create_release_table(name, ads_client)
+      query = 'CREATE TABLE IF NOT EXISTS "LCM_RELEASE" (segment_id VARCHAR(255) NOT NULL, master_project_id VARCHAR(255) NOT NULL, version INTEGER NOT NULL, maql_ddl VARCHAR(65000));'
+      query.gsub!('LCM_RELEASE', name)
+      ads_client.execute(query)
+    end
+
+    def create_workspace_table(name, ads_client, client_id_column = 'client_id')
+      query = 'CREATE TABLE IF NOT EXISTS "LCM_WORKSPACE" (client_id VARCHAR(255) NOT NULL, segment_id VARCHAR(255) NOT NULL, project_id VARCHAR(255), project_title VARCHAR(255) NOT NULL);'
+      query.gsub!('LCM_WORKSPACE', name)
+           .gsub!('client_id', client_id_column)
+      ads_client.execute(query)
+    end
+
+    def create_suffix
```

Comment:
Similar code found in shared_contexts_for_lcm_specs.rb:7, configuration_helper.rb:7, lcm_helper.rb:16 (mass = 90)